src/lt/object.cljs

```diff
@@ -285,12 +287,12 @@
   [obj & r]
   (swap! obj #(apply update-in % r)))
 
-(defn assoc-in! [obj k v]
+(defn- assoc-in! [obj k v]
```

Comment:
Agree update-in is the better name. One day we should deprecate update!. Will change assoc-in!